packages/big-design/src/components/Dropdown/types.ts

```diff
@@ -2,25 +2,25 @@ import { Placement } from 'popper.js';
 
 import { ListItemProps } from '../List/Item';
 
-export interface DropdownProps extends Omit<React.HTMLAttributes<HTMLUListElement>, 'children'> {
+export interface DropdownProps<T> extends Omit<React.HTMLAttributes<HTMLUListElement>, 'children'> {
   maxHeight?: number;
-  options: Array<DropdownItem | DropdownLinkItem>;
+  options: Array<Item<T> | LinkItem<T>>;
   placement?: Placement;
   trigger: React.ReactElement;
 }
 
-interface BaseItem extends Omit<ListItemProps, 'children' | 'content' | 'onClick' | 'value'> {
+interface BaseItem<T> extends Omit<ListItemProps, 'children' | 'content' | 'onClick' | 'value'> {
   content: string;
   icon?: React.ReactElement;
-  value?: any;
-  onClick?(item: DropdownItem | DropdownLinkItem): void;
+  value?: T;
+  onClick?(item: Item<T> | LinkItem<T>): void;
 }
 
-export interface DropdownItem extends BaseItem {
+export interface Item<T> extends BaseItem<T> {
```

Comment:
Is there a reason we renamed this to `Item` instead of `DropdownItem`? `Item` is too generic IMO. Same for `LinkItem`.